openml/testing.py

```diff
@@ -35,11 +34,11 @@ class TestBase(unittest.TestCase):
     # amueller's read/write key that he will throw away later
     apikey = "610344db6388d9ba34f6db45a3cf71de"
 
-    # creating logger for unit test file deletion status
-    logger = logging.getLogger("unit_tests")
-    logger.setLevel(logging.INFO)
-    fh = logging.FileHandler('TestBase.log')
-    fh.setLevel(logging.INFO)
+    # creating logger for tracking files uploaded to test server
+    logger = logging.getLogger("unit_tests_publish")
+    logger.setLevel(logging.DEBUG)
+    fh = logging.FileHandler('UploadedFiles.log')
+    # fh.setLevel(logging.DEBUG)
```

Comment:
Is it actually necessary to introduce a logger which logs to a file? Also, could you please remove that comment?